_inc/client/plans/plan-body.jsx

```diff
@@ -313,7 +313,9 @@ const PlanBody = React.createClass( {
 						</div>
 
 						<p>
-							<Button href={ 'https://jetpack.com/redirect/?source=plans-main-bottom&site=' + this.props.siteRawUrl } className="is-primary">
+							<Button href={ 'jetpack_free' === this.props.plans
```

Comment:
This is handled in the parent `plans/index.jsx` and `plans` prop will be empty if the reducer `getSitePlan` in `state/site` returns an empty object, which is what it returns as default when there is no data plan.
That said I've added a default and set the prop type in https://github.com/Automattic/jetpack/pull/6445/commits/126bdcb5a665aef94b2b5dae6af24a5fceb1876b